### PR TITLE
Update logLevel help message and related doc etc

### PR DESCRIPTION
**Pull request by jiacheliu3**
This addresses some missing points in https://github.com/Alluxio/alluxio/commit/0c72e4e63688dbee776b24139f86cf2b495150cd
1. Add relevant doc mentioning the job master and job workers are now supported
2. Improve the relevant doc
3. When `--target` is not specified, all targets should include the job master and job workers
4. Identified a few issues in the `logLevel` command. Added TODOs for further PRs to resolve.

### CODE REVIEW

File: docs/en/operation/Basic-Logging.md [hunk 1]
```diff
@@ -86,16 +86,7 @@ runtime.
...
-
-The command follows the format `alluxio logLevel --logName=NAME [--target=<master|worker|host:port>] [--level=LEVEL]`,
-where:
-* `--logName <arg>` indicates the logger's class (e.g. `alluxio.master.file.DefaultFileSystemMaster`)
-* `--target <arg>` lists the Alluxio master or workers to set.
-The target could be of the form `<master|workers|host:webPort>` and multiple targets can be listed as comma-separated entries.
-The `host:webPort` format can only be used when referencing a worker.
-The default target value is all masters and workers.
-* `--level <arg>` If provided, the command changes to the given logger level,
-otherwise it returns the current logger level.
```
Comment (jiacheliu3) [inline docs/en/operation/Basic-Logging.md:98]: Replaced duplicated session with a link

File: docs/en/operation/Basic-Logging.md [hunk 2]
```diff
@@ -123,10 +114,10 @@ Furthermore, you can turn on Alluxio debug logging when you are troubleshooting
 in a running cluster, and turn it off when you are done.
 ```console
 # Turn on Alluxio debug logging and start debugging
-$ ./bin/alluxio logLevel --logName=alluxio --target=master,workers --level=DEBUG
```
Comment (jiacheliu3) [inline docs/en/operation/Basic-Logging.md:126]: Now the valid targets is no longer just `master,workers`. By not specifying the target, all master/job_master/workers/job_workers are updated.

File: shell/src/main/java/alluxio/cli/LogLevel.java [hunk 3]
```diff
@@ -196,6 +196,7 @@ public static void logLevel(String[] args, AlluxioConfiguration alluxioConf)
           targetInfoList.add(jobWorker);
         }
       } else if (target.contains(":")) {
+        // TODO(jiacheng): Support all other roles, not just worker
```
Comment (jiacheliu3) [inline shell/src/main/java/alluxio/cli/LogLevel.java:199]: The original command only considers the manually specified targets to be a worker. I think this should support all possible roles, not limited to workers and job-workers. The change will not be trivial so I intend to do it in another PR, to keep the review cycle short.

Comment (jiacheliu3) [reply shell/src/main/java/alluxio/cli/LogLevel.java:199]: Ongoing changes can be found in https://github.com/Alluxio/alluxio/pull/12874/files

File: docs/en/operation/User-CLI.md [hunk 4]
```diff
@@ -208,29 +208,18 @@ Status: CANCELED
...
-```console
-$ ./bin/alluxio logLevel --logName=alluxio.heartbeat.HeartbeatContext \
-  --target=master,192.168.100.100:30000 --level=DEBUG
-```
-
-And the following command returns the log level of the class `alluxio.heartbeat.HeartbeatContext` among all the workers:
-```console
-$ ./bin/alluxio logLevel --logName=alluxio.heartbeat.HeartbeatContext \
-  --target=workers
-```
```
Comment (jiacheliu3) [inline docs/en/operation/User-CLI.md:233]: Same, remove duplicated section

File: docs/en/operation/User-CLI.md [hunk 5]
```diff
@@ -208,29 +208,18 @@ Status: CANCELED
...
-The command follows the format `alluxio logLevel --logName=NAME [--target=<master|worker|host:port>] [--level=LEVEL]`,
+The command follows the format `alluxio logLevel --logName=NAME [--target=<master|workers|job_master|job_workers|host:port>] [--level=LEVEL]`,
 where:
 * `--logName <arg>` indicates the logger's class (e.g. `alluxio.master.file.DefaultFileSystemMaster`)
 * `--target <arg>` lists the Alluxio master or workers to set.
-The target could be of the form `<master|workers|host:webPort>` and multiple targets can be listed as comma-separated entries.
+The target could be of the form `<master|workers|job_master|job_workers|host:webPort>` and multiple targets can be listed as comma-separated entries.
 The `host:webPort` format can only be used when referencing a worker.
-The default target value is all masters and workers.
+The default target value is the master, job master, all workers and job workers.
```
Comment (jiacheliu3) [inline docs/en/operation/User-CLI.md:217]: This is tricky, I think currently it only reads the local `alluxio.master.hostname` and works with one master.
When master is HA, it does NOT try to find the primary master. Added a TODO but not resolving in this PR to keep it simple.

File: shell/src/main/java/alluxio/cli/LogLevel.java [hunk 6]
```diff
@@ -236,7 +237,7 @@ private static void setLogLevel(final TargetInfo targetInfo, String logName, Str
     if (level != null) {
       uriBuilder.addParameter(LEVEL_OPTION_NAME, level);
     }
-    LOG.info("Setting log level on %s%n", uriBuilder.toString());
+    LOG.info("Setting log level on {}", uriBuilder.toString());
```
Comment (jiacheliu3) [inline shell/src/main/java/alluxio/cli/LogLevel.java:241]: small fix

File: docs/en/operation/Basic-Logging.md [hunk 7]
```diff
@@ -123,14 +115,12 @@ Furthermore, you can turn on Alluxio debug logging when you are troubleshooting
...
+$ ./bin/alluxio logLevel --logName=alluxio --level=DEBUG
 
 # Turn off Alluxio debug logging when you are done
-$ ./bin/alluxio logLevel --logName=alluxio --target=master,workers --level=INFO
+$ ./bin/alluxio logLevel --logName=alluxio --level=INFO
 ```
 
-For more information, refer to the help text of the `logLevel` command by running `./bin/alluxio logLevel`
-
```
Comment (jiacheliu3) [inline docs/en/operation/Basic-Logging.md:133]: Running this command does not really provide any more information.

File: docs/en/operation/User-CLI.md (hunk 5 above)
Comment (jiacheliu3) [reply docs/en/operation/User-CLI.md:217]: Logged in https://github.com/Alluxio/alluxio/issues/12889

File: docs/en/operation/User-CLI.md [hunk 8]
```diff
@@ -208,29 +208,20 @@ Status: CANCELED
 The `logLevel` command returns the current value of or updates the log level of a particular class
 on specific instances. Users are able to change Alluxio server-side log levels at runtime.
 
-The command follows the format `alluxio logLevel --logName=NAME [--target=<master|worker|host:port>] [--level=LEVEL]`,
+The command follows the format `alluxio logLevel --logName=NAME [--target=<master|workers|job_master|job_workers|host:port>] [--level=LEVEL]`,
```
Comment (LuQQiu) [inline docs/en/operation/User-CLI.md:211]: The target is the leading master, leading job_master?
maybe we can include this information (only change the leading master/job_master) somewhere

File: docs/en/operation/User-CLI.md [hunk 9]
```diff
@@ -208,29 +208,20 @@ Status: CANCELED
...
 where:
 * `--logName <arg>` indicates the logger's class (e.g. `alluxio.master.file.DefaultFileSystemMaster`)
 * `--target <arg>` lists the Alluxio master or workers to set.
-The target could be of the form `<master|workers|host:webPort>` and multiple targets can be listed as comma-separated entries.
+The target could be of the form `<master|workers|job_master|job_workers|host:webPort>` and multiple targets can be listed as comma-separated entries.
 The `host:webPort` format can only be used when referencing a worker.
-The default target value is all masters and workers.
+The default target value is the local master, local job master, all workers and job workers.
```
Comment (LuQQiu) [inline docs/en/operation/User-CLI.md:217]: ??? local master, local job master instead of the leading master, leading job_master?

**Review by LuQQiu — COMMENTED** [review]
Confusing on what the master/job_master stand for, otherwise LGTM

**Review by LuQQiu — APPROVED** [review]

**Comment by jiacheliu3** [comment]
alluxio-bot, merge this please
